lib/provider/camunda/element-templates/parts/CustomProps.js

```diff
@@ -609,6 +624,20 @@ function setPropertyValue(element, property, value, bpmnFactory) {
     ));
   }
 
+  if (bindingType === CAMUNDA_FIELD) {
+    var existingFieldInjections = findExtensions(bo, [ 'camunda:Field' ]);
+    var newFieldInjections = createCamundaFieldInjection(binding, value, bpmnFactory);
+    updates.push(cmdHelper.addAndRemoveElementsFromList(
+      element,
+      extensionElements,
+      'values',
+      null,
+      [ newFieldInjections ],
+      existingFieldInjections ? [ existingFieldInjections ] : []
```

Comment:
Shouldn't this be `existingFieldInjections ? [ existingFieldInjections ] : []`? There is definitely a test case missing here.